internal/cli/cmd/cluster/create.go

```diff
@@ -142,6 +140,7 @@ var clusterCreateExample = templates.Examples(`
 const (
 	CueTemplateName = "cluster_template.cue"
 	monitorKey      = "monitor"
+	saNamePrefix    = "kb-sa-"
```

Comment:
keep consistent with the SA name in `internal/controller/builder/cue/rbac_template.cue`